pkg/network/dns/driver_windows.go

```diff
@@ -16,10 +16,9 @@ import (
 	"time"
 	"unsafe"
 
+	"github.com/DataDog/datadog-agent/pkg/network/driver"
```

Comment:
I think it's technically right.  Alphabetically.

Reply:
I have seen other companies have tree section in imports: global, github, private. I thought that Datadog Agent code did it as well, hence my comment.

Reply:
We do, but we don't enforce it with an automatic linter, since that didn't work very well.  See https://github.com/DataDog/datadog-agent/blob/main/docs/dev/imports.md